File: Folder Python/Elektronik.py
from Produk import Produk

class Elektronik(Produk):
    def __init__(self, kode, nama, harga, merek, garansi, daya):
        super().__init__(kode, nama, harga)
        self.merek = merek
        self.garansi = garansi
        self.daya = daya

    def getMerek(self):
        return self.merek

    def getGaransi(self):
        return self.garansi

    def getDaya(self):
        return self.daya

File: Folder Java/Elektronik.java
public class Elektronik extends Produk {
    protected String merek, garansi;
    protected int daya;

    public Elektronik(String kode, String nama, int harga,
                      String merek, String garansi, int daya) {
        super(kode, nama, harga);
        this.merek = merek;
        this.garansi = garansi;
        this.daya = daya;
    }

    public String getMerek() { return merek; }
    public String getGaransi() { return garansi; }
    public int getDaya() { return daya; }
}